content/2018-03-21_pracuj-s-nami.md

```diff
@@ -0,0 +1,69 @@
+Title: Pracuj s námi
+Date: 2018-03-21 17:00:00
+
+
+Pracuji pro [Apiary](https://apiary.io/), což je bývalý český startup. Vytváříme nástroje pro návrh webových API. Píšu bývalý, protože jsme [startup prodali](https://byznys.ihned.cz/c1-65593630-oracle-kupuje-za-miliardy-korun-cesky-start-up-apiary-zakladatele-ve-firme-zustavaji). Původní tým ale zůstává a naopak by potřeboval doplnit. Jako startup jsme stále balancovali na hraně možností, sháněli investice, cíle se měnily z měsíce na měsíc. Být pod [Larryho](https://en.wikipedia.org/wiki/Larry_Ellison) křídly sice taky přináší výzvy, ale v zásadě máme konečně možnost nabrat lidi, opravit co je rozbité, a vytvořit vše, co jsme si kdy vysnili.
+
+V Apiary vedu malý tým, který zodpovídá za všechny nástroje kolem testování API. Jedním z nich je třeba [Dredd](https://github.com/apiaryio/dredd) s více jak 2000 [hvězdičkami na GitHubu](https://github.com/thepracticaldev/orly-full-res/blob/master/choosingbasedongithubstars-big.png). Snažíme se být [Open Source](https://opensource.guide/) _first_, testovat jak se má, psát dokumentaci jak se má.
+
+![Dredd]({filename}/images/dredd.png)
+
+Máme velké plány, které ovšem ve dvou nezvládneme. A protože jde nejen o nástroje do terminálu, ale i o webové aplikace, které poběží uživatelům pod rukama přímo na apiary.io, v současném složení týmu to nezvládneme už vůbec. Zatímco [Michal](https://github.com/michalholasek) je renesanční osobnost a všeumětel (tedy _full stack_), tak já, jakožto [přeběhlík z Pythonu]({filename}/2014-05-29_proc-python.md) do Node.js, jsem na frontend úplný lempl.
+
+![Honza a Michal]({filename}/images/sjs-2017.png)
+
+## Koho hledáme?
+
+**Hledáme:** frontend, nejlíp React, spíše zkušenější, full time, spíš Praha<br>
+**Plný popis:** [tady na LinkedIn][linkedin] (to jsme napsali spolu s Michalem, není to někde z hlubin HR)
+
+Máte-li otázky, klidně pište přímo nám dvěma. Jinak prosím rovnou Monice na [monika.mikesova@oracle.com](mailto:monika.mikesova@oracle.com).
+
+## Výjevy ze života Apiary
+
+Nebudu nic zastírat, jsme startup, který koupila korporace. Každý startup je jiný, každá korporace, každá akvizice. Nikdo kromě nás nemá jasnou představu o tom, jak to ve skutečnosti dopadlo, takže mohou kolovat nejrůznější předsudky a mýty. Proto poskytuji následující _nakoukání do Apiary kultůry_:
+
+![Akvizice]({filename}/images/putin.jpg)
+Akvizicí začalo naše dobrodružství s Oracle. Dostali jsme se na hlavní stránku novin, spolu s Putinem v ponorce. Kdo z vás to má?
```

Comment:
Vis o tom, ze ten clanek je o investici a ne o akvizici, zejo? ;)

Reply:
Eh! Se mi to úplně smotalo dohromady :D Napravím. Dík.